src/app.py

```diff
@@ -46,6 +46,24 @@ def index():
     return render_template("index.html", affiliations=affiliations_set, email=email)
 
 
+@app.route("/edit")
+def edit():
+    """The edit route.
+
+    Edit an existing affiliation.
+    """
+    logger.info("User accessed edit")
+    affil_id = request.args.get('affil')
+    if not affil_id:
+        return redirect(url_for("index"))
+    email = session["email"] if "email" in session else None
+    # TODO(sanchegm): Redirect to main page if user is not logged in.
```

Comment:
What about doing something like [this](https://stackoverflow.com/a/25233898/15027348)?